ui/src/app/applications/components/application-details/application-resource-filter.tsx

```diff
@@ -105,6 +123,15 @@ export const Filters = (props: {
                 }))
             })}
             {namespaces.length > 1 && ResourceFilter({label: 'NAMESPACES', prefix: 'namespace', options: (namespaces || []).filter(l => l && l !== '').map(toOption), field: true})}
+            {/* {(tree.orphanedNodes || []).length > 0 &&  */}
```

Comment:
Please do show it!
  - I've sometimes needed to untick it before performing an action that will create orphans
  - I assume the filters are still "global"? -> sometimes I have had an application with *thousands* of orphans which crashes the UI when I look at it: I have to go to another application to toggle the filter.
  - It can be annoying to have it appear/disappear as orphans are added/removed.